.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@ name: Test
 "on":
   push:
     branches:
-      - master
+      - main
```

Comment:
- Move from `master` to `main` on this fork
- `develop` may be used to keep two repos in sync until moving to `main`